terraformutils/hcl.go

```diff
@@ -92,6 +92,8 @@ func (v *astSanitizer) visitObjectItem(o *ast.ObjectItem) {
 			t.Token.Type = 10
 			// check if text json for Unquote and Indent
 			jsonTest := t.Token.Text
+			// Since it's heredoc, we probably want to unescape quotes no matter what
+			t.Token.Text = strings.ReplaceAll(jsonTest, "\\\"", "\"")
```

Comment:
we change t.Token.Text after this in line https://github.com/GoogleCloudPlatform/terraformer/pull/1011/files#diff-f78fda63c41054c31898033695a0a3d064a48bf5a3a1e558a54f42bbd4a0b5a0R116

Reply:
@sergeylanzman yeah, if the text is valid json, then `t.Token.Text` will be set to the unquoted / indented pretty-formated json string. But if it fails json parsing (e.g. if the text was hcl text) then we're left with escaped quotation marks. 

Maybe the better place to put this unescaping in the `err != nil` block to make it more clear that it only really matters when the json parsing fails?